scripts/sql/108_create_semgrep_plugin.down.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM plugin_step_variable WHERE name = 'SemgrepAppToken';
+DELETE FROM plugin_step_variable WHERE name = 'PrefixAppNameInSemgrepBranchName';
+DELETE FROM plugin_step_variable WHERE name = 'UseCommitAsSemgrepBranchName';
+DELETE FROM plugin_step_variable WHERE name = 'SemgrepAppName';
+DELETE FROM plugin_step_variable WHERE name = 'ExtraCommandArguments';
```

Comment:
can we add default value for this `--json`

Reply:
added --json in semgrep ci command